app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    original_file = command.split()[1]
+    new_file = command.split()[2]
```

Comment:
you can do it like this
```suggestion
    command_name, base_filename, new_filename = command.split()
```